taca/server_status/cronjobs.py

```diff
@@ -9,35 +9,34 @@
 
 from taca.utils.config import CONFIG
 
-def update_cronjob_db():
-    logging.info('Getting list of cronjobs')
-    crontab = CronTab()
-    # CronTab() is supposed to return the list of cronjobs
-    # it works on milou, but doesn't work on preproc (returns an empty list)
-    # when it doesn't work, create CronTab object from the output of command 'crontab -l'
-    if crontab.crons == []:
-        try:
-            p = subprocess.Popen('crontab -l', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        except Exception, e:
-            logging.error(e.message)
-        else:
-            output = p.communicate()[0]
-            crontab = CronTab(tab=output)
+def _parse_crontab():
+    result = {}
+    user = getpass.getuser()
+    logging.info('Getting crontab for user {}'.format(user))
+    try:
+        crontab = CronTab(user=user)
```

Comment:
yeah, that's my guess, I haven't read the crontab module, but the errors means that.